post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -27,6 +25,13 @@ include::modules/machine-config-node-drain.adoc[leveloffset=+2]
 * xref:../support/troubleshooting/troubleshooting-operator-issues.adoc#troubleshooting-disabling-autoreboot-mco_troubleshooting-operator-issues[Disabling the Machine Config Operator from automatically rebooting]
 
 include::modules/machine-config-drift-detection.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* For more information about detecting configuration drift, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection].
```

Comment:
```suggestion
* xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-drift-detection_post-install-machine-configuration-tasks[Understanding configuration drift detection]
```
Do not include text outside of the square brackets in additional resources: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections